### PR TITLE
Issue 7 portfolio table in db

### LINKED ISSUE #7: portfolio table in db
All shares should be attached to a user portfolio. This portfolio should show what stocks a user owns

### LINKED ISSUE #15: Portfolio Class
This class should contain all information on a users portfolio. The SQL table should contain all the information necessary for when a user purchases a specific share. What share? how much? how many? who? date purchased?
- userID
- Ticker Name
- Ticker price
- Ticker shares owned

### PR DESCRIPTION
**Pull request by zhowey**
closes #18 
closes #15 
closes #7

### CODE REVIEW

File: stock_sim/modules/dbWriter.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import mysql.connector
 import os, sys
 from datetime import datetime
-
```
Comment (killkelleyr) [inline stock_sim/modules/dbWriter.py]: For future, i woul leave a space between you imported libraries and your classs

File: stock_sim/modules/dbWriter.py [hunk 2]
```diff
@@ -13,30 +12,26 @@ def __init__(self):
                 'host': os.environ['DB_HOST'],
                 'database': 'stocksimdb'
                 }
-            self.stocksimdb = mysql.connector.connect(**stocksim_config)
+            self.stocksimdb = mysql.connector.connect()
```
Comment (killkelleyr) [inline stock_sim/modules/dbWriter.py]: Thse should not be changed on dbWriter, please revert back to what was there previously

File: stock_sim/modules/dbWriter.py [hunk 3]
```diff
@@ -47,9 +42,28 @@ def updateUserLogin(user_id):
...
                      WHERE user_id = "{}"
                      """).format(cxn.currentTime(),user_id)
-                     
         cursor=cxn.stocksimdb.cursor()
         cursor.execute(statement)
         cxn.stocksimdb.commit()
         cursor.close()
-        cxn.stocksimdb.close()
\ No newline at end of file
+        cxn.stocksimdb.close()
```
Comment (killkelleyr) [inline stock_sim/modules/dbWriter.py:53]: please add a new line between a new method

**Review by killkelleyr — CHANGES_REQUESTED** [review]
Please review comments and make a few modifications

File: stock_sim/modules/dbWriter.py (hunk 3 above)
Comment (zhowey) [reply stock_sim/modules/dbWriter.py:53]: edited

File: stock_sim/modules/dbWriter.py (hunk 2 above)
Comment (zhowey) [reply stock_sim/modules/dbWriter.py]: edited

File: stock_sim/modules/dbWriter.py (hunk 1 above)
Comment (zhowey) [reply stock_sim/modules/dbWriter.py]: edited

**Review by killkelleyr — APPROVED** [review]

**Pull request merged**